src/index.html

```diff
@@ -8,20 +8,26 @@
   <script src="https://kit.fontawesome.com/bd260a0fea.js" crossorigin="anonymous"></script>
 </head>
 <body>
+  
   <section>
-<div id="todo-list"><h1>To-Do List<i class="fa-solid fa-arrows-rotate"></i></h1>
+    <div id="todo-list">
+      <h1>To-Do List<i class="fa-solid fa-arrows-rotate"></i></h1>
     
-    <input type="text" id="new-task" placeholder="Add a task ...">
-      <i class="fa-solid fa-arrow-turn-down fa-rotate-90"></i>
 
-    <div class="me-1">
-    <ul id="task-list" class="drag-list"></ul>
+      <div class="add-task-wrapper">
+        <input type="text" id="new-task" placeholder="Add a task ...">
+          <button type="submit" id="add-task-btn">Add Task</button>
+      </div>
+
+
+      <div class="me-1">
+        <ul id="task-list" class="drag-list"></ul>
+      </div>
```

Comment:
- [ Optional ] I think it would be nice if your design for editing a `todo` is as close as possible to the design in the video provided for this milestone. Your design has a `prompt` dialogue box that can be used to edit a `todo` item whereas in the project video, the editing of todos is done in the `todolist` using an `input element`. I would like to suggest that you implement this functionality without using a `prompt` dialogue because different browsers have different appearances for the built-in dialogues. Please see the screenshots below for more details on this and rectify it.

## Your design
![screenshot](https://github.com/efmomoh/Webpack-demo/assets/16392046/f59b1aa5-d2c7-4e7b-96ee-827af0cf6bcd)

## Project Video
https://www.youtube.com/watch?v=AcUd-_Yjjqg